draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -242,30 +242,19 @@ is typically done with delays below an RTT; thus, rule #2 in {{rules}} should co
 ### Specification
 
 {{Section 5.1 of !RFC4341}} states:
->    There are currently no standards governing TCP's use of the
-   congestion window during an application-limited period.  In
-   particular, it is possible for TCP's congestion window to grow quite
-   large during a long uncongested period when the sender is application
-   limited, sending at a low rate.  [RFC2861] essentially suggests that
-   TCP's congestion window not be increased during application-limited
-   periods when the congestion window is not being fully utilized.
+>There are currently no standards governing TCP's use of the congestion window during an application-limited period.  In particular, it is possible for TCP's congestion window to grow quite large during a long uncongested period when the sender is application limited, sending at a low rate.  {{?RFC2861}} essentially suggests that TCP's congestion window not be increased during application-limited periods when the congestion window is not being fully utilized.
```

Comment:
/quite large/arbitrarily large/ ?
/application limited/rate-limited/ as per RFC 7661 ?
/low rate/rate lower than permitted by the cwnd/
/RFC2861/RFC7661/
/not be/is not/
/application limited/rate-limited/ as per RFC 7661 ?
/the congestion window is not being fully utilized./the sender if not fully utilising the congestion window./

Reply:
All of this is a literal quote from RFC 4341!

Reply:
See a short PR that tries to undo some of the legacy of re41 and 2861. Note 2861 can't be normative.